scripts/importer.py

```diff
@@ -73,7 +77,8 @@ def main():
 
     import_function = {
         'netscape': import_netscape_bookmarks,
-        'mozilla': import_moz_places
+        'mozilla': import_moz_places,
+        'chrome': import_chrome
```

Comment:
I didn't notice this before, but please add a trailing comma here and in `browser_default_input_format`.